### PR TITLE
added docker compose stack and github action

### PR DESCRIPTION
**Pull request by attwad**
In which github, running as docker containers, starts action workflows as docker containers which download docker containers and run stuff in them. We need to go deeper. And I need to regain my sanity.

### CODE REVIEW

**Comment by attwad** [comment]
[docker-compose setup for dev environment](https://trello.com/c/55tz7mmc/41-docker-compose-setup-for-dev-environment)

File: dev/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+During development, you can run each service individually using `docker` or plug them together using `docker-compose`.
+
+`docker-compose` is also used by Github actions to make sure services are working well together all the time.
+
+### All services composed with hot reload
+
+From this folder run:
+
+```
+docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build
```
Comment (axmb) [inline dev/README.md:16]: Worth making a script for this? At risk of exposing the magnitude of my own laziness...

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: Docker Compose Actions Workflow
+on: [push, pull_request]
```
Comment (axmb) [inline .github/workflows/docker.yml:2]: Missed this, sorry -- do we want to restrict at least push to `master`?

File: dev/README.md (hunk 1 above)
Comment (attwad) [reply dev/README.md:16]: absolutely, done

File: .github/workflows/docker.yml (hunk 2 above)
Comment (attwad) [reply .github/workflows/docker.yml:2]: "push" is the only way to test workflows afaict, I don't think we want to restrict to master

File: dev/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        ports:
+            - "27017:27017"
+    curator:
+        build: ../verification/curator-service
+        ports:
+            - "3001:3001"
+        depends_on:
+            - mongo
+        environment:
+            DB_CONNECTION_STRING: "mongodb://mongo:27017/test"
```
[the remark this answers is not in the document]
Comment (attwad) [reply dev/docker-compose.yml:15]: yep, done.

**Pull request merged**